src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Mapping/ToViewModel/ReturnAndSchemeDataToReceivedPcsViewModelMapTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.AatfReturn.Mapping.ToViewModel
+{
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.Scheme;
+    using EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel;
+    using EA.Weee.Web.Services.Caching;
+    using FakeItEasy;
+    using FluentAssertions;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class ReturnAndSchemeDataToReceivedPcsViewModelMapTests
+    {
+        private ReturnAndSchemeDataToReceivedPcsViewModelMap mapper;
+
+        public ReturnAndSchemeDataToReceivedPcsViewModelMapTests()
+        {
+            mapper = new ReturnAndSchemeDataToReceivedPcsViewModelMap(A.Fake<IWeeeCache>(), new TonnageUtilities());
```

Comment:
We should really use the ITonnageutilities rather than concrete type and fake calls to the class...